docker/README.md

```diff
@@ -38,13 +38,20 @@ References
 https://github.com/hypertrace/hypertrace/issues/114
 https://github.com/hypertrace/hypertrace/issues/124
 
-#### To start alerting related services
+### To start alerting related services
 
 Run ```docker-compose -f docker-compose.yml  -f docker-compose.alerting.yml up```
 or ```docker-compose -f docker-compose.yml -f docker-compose.postgres.yml -f docker-compose.alerting.yml up``` (for postgres as document-store)
 
 Update alerting rule definition in alert-rules.json, notification-rules.json
 
+### Enable metrics pipeline components
```

Comment:
user also have to override the query service config to read from prometheus?

Reply:
Can we extend that once QS part all merged?

Reply:
yeah sure